src/lib/tabs/tab-group.spec.ts

```diff
@@ -295,6 +294,26 @@ describe('MdTabGroup', () => {
     });
   });
 
+  describe('with 100% height tab content', () => {
+    let fixture: ComponentFixture<TabGroupWithStretchedHeight>;
+    let tabGroup: MdTabGroup;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(TabGroupWithStretchedHeight);
+      fixture.detectChanges();
+
+      tabGroup =
+          fixture.debugElement.query(By.directive(MdTabGroup)).componentInstance as MdTabGroup;
+    });
+
+    it('should grow to height of the tab group', () => {
+      const tabContent = fixture.componentInstance.content.nativeElement;
+
+      // Check that the content is greater than 0 (it became larger to match the tab body height)
+      expect(tabContent.getBoundingClientRect().height).toBeGreaterThan(0);
```

Comment:
Instead of a comment, you could pass the info via `toBeGreaterThan`. That way, if it fails, it'll be a bit easier to see what's going on:

```ts
expect(height).toBeGreaterThan(0, 'Expected the tab content to have a height.');
```